src/agent-server.ts

```diff
@@ -23,7 +23,7 @@ export function* createAgentServer(orchestrator: Execution, options: AgentServer
     [message] = yield on(child, "message");
   } while(message.type !== "ready");
 
-  orchestrator.send({ ready: "agent" });
+  yield send({ ready: "agent" }, orchestrator);
```

Comment:
Like `receive`, send takes the context to receive the message as its second argument.